### PR TITLE
BETA: Register darvingaba.is-a.dev

### PR DESCRIPTION
**Pull request by darvingaba**
Added `darvingaba.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/darvingaba.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "darvingaba",
+        "email": "darvingaba1@gmail.com"
+    },
+    "record": {
+        "URL": "https://portfolio-lac-psi-77.vercel.app/"
```
Comment (Stef-00012) [inline domains/darvingaba.json:7]: I cannot reach your website, vercel says
> 404: NOT_FOUND
> Code: DEPLOYMENT_NOT_FOUND
> ID: fra1::lm9t4-1707408132681-de49b9a1f423
> 
> [This deployment cannot be found. For more information and troubleshooting, see our documentation.](https://vercel.com/docs/errors/platform-error-codes#deployment_not_found)

**Review by Stef-DP — CHANGES_REQUESTED** [review]

**Comment by darvingaba** [comment]
added website link

File: package-lock.json
Comment (Stef-00012) [inline package-lock.json:1]: You can't add files to the root folder

**Review by Stef-DP — CHANGES_REQUESTED** [review]

**Comment by Stef-00012** [comment]
seems good now

**Review by iostpa — APPROVED** [review]
seems good to me

**Pull request merged**